### ISSUE TITLE
minify failure

### ISSUE THREAD

**Issue by dohvis** [wilsonzlin/minify-html#31]
```python
from minify_html import minify
text = '<meta content="</HEADER><BODY><BODY></HTML>aaaa" >'

minify(text)
# actual: <meta content=</HEADER><BODY><BODY></HTML>aaaa>
# expected: <meta content="</HEADER><BODY><BODY></HTML>aaaa">
```

**Comment by wilsonzlin** [comment]
Thanks for raising, I've just fixed this in the latest version `0.4.7`, let me know if it fixes it for you.

**Comment by dohvis** [comment]
It fixed! Thank you :)

For history: https://github.com/wilsonzlin/minify-html/commit/df6426cb251e71073a0e21e6b81f0a3cc5bb3c8f
